clients/web/src/lib/components/atoms/ProgressBar/types.ts

```diff
@@ -0,0 +1,3 @@
+export type ProgressBarVariant = "info" | "primary";
+export type ProgressBarSize = "small" | "default" | "large" | "extraLarge";
```

Comment:
For the form elements, I've been using `sm`, `md`, `lg`, and `xl` for the sizes to make it a little less verbose. `md` is the default value. Lets standardize to that.